### DISCUSSION TITLE
Tutorials for BO and EA-BO methods

### DISCUSSION THREAD

**Discussion by harinduRM** [Tomoki-YAMASHITA/CrySPY#75] [Q&A]
Hello,
Going through the tutorials page I have noticed that currently there are no Tutorials for BO and EA-BO methods.
Could you kindly provide any materials you have on this as I am a beginner to this package and would greatly benefit from having some tutorials in hand?

**Comment by Tomoki-YAMASHITA** [comment]
OK. I'll write the tutorials pages for BO and EA-BO.
But, I’m currently preparing for the release of the next version, so I might be able to do it afterward.
